src/string-parser/parse-call.ts

```diff
@@ -0,0 +1,38 @@
+import { parseSuit } from '.';
+import { Types } from '..';
+import { AuctionCall, NoTrumpType, PossibleCalls } from '../types';
+
+/**
+ * Checks strain of bid for NT or a suit
+ * @param str Letter denoting suit or NT
+ * @returns NT or Suit type
+ */
+function parseStrain(str: string): Types.Suit | NoTrumpType {
+	return str === 'n' || str === 'N' ? Types.NoTrump : parseSuit(str);
+}
+
+/**
+ * Parses a single call in an auction. It can either be a bid
+ * @param bid A string of the call to be parsed.
+ * @returns AuctionCall.call object or an error if it can't be parsed
+ */
+
+export function parseCall(bid: string): AuctionCall {
+	if (!Number.isNaN(Number(bid[0]))) {
```

Comment:
This doesn't really check a whole lot, [see here](https://thisthat.dev/number-constructor-vs-parse-int/). It's probably better to just switch to `parseInt`. I'd also only do the parsing once and save it to a variable.